server/vulnerabilities/db.go

```diff
@@ -21,37 +21,57 @@ func sqliteDB(dbPath string) (*sqlx.DB, error) {
 }
 
 func applyCPEDatabaseSchema(db *sqlx.DB) error {
+	// Use a new table cpe_2 containing new columns vendor, product. view cpe used for backwards compatibility
```

Comment:
wondering if there's a good way to add an automated test for backwards compatibility, but not a big deal.

Reply:
Added a test, and found an issue. Thanks for the suggestion!